src/D_functions_and_procedures.sql

```diff
@@ -428,7 +444,7 @@ BEGIN
 	FOREACH actor IN ARRAY COALESCE(actors, '{}')
 	LOOP
 		INSERT INTO similar_cast(media_id, "count")
-		SELECT DISTINCT cm.media_id, 1
+		SELECT cm.media_id, 1
```

Comment:
what is the point of selecting `1` here?

Reply:
missed that. this does not need to be there